src/validator/validationManager.ts

```diff
@@ -69,13 +69,26 @@ export class ValidationManager {
     if (typeof result == 'string') {
       return result;
     }
-
     return true;
   }
 
   public async validateUpdate(identifier: string, payload: UpdatePayload): Promise<boolean | string> {
     let result: boolean | string;
 
+    if (payload.footprint != undefined) {
+      result = this.validateFootprint(payload.footprint);
+      if (typeof result == 'string') {
+        return result;
+      }
+    }
+
+    if (payload.sourceDateEnd && payload.sourceDateStart != undefined) {
```

Comment:
What happens if the client provides only one date? You need to handle this option too.
If the client provided one date, you should get the other date from DB and call the function ValidateDates with the provided date and the other date from DB